tests/v1/FitnessDemo3/run_test

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+error_reporting(E_ALL);
+
+$PWD = __DIR__;
+
+// Must load autoloader before including any Annotation Processors
+if (file_exists($autoloaderFilePath = dirname($PWD, 5) . '/autoload.php')) {
+    /** @noinspection PhpIncludeInspection */
+    require_once $autoloaderFilePath;
+} else if (file_exists($autoloaderFilePath = dirname($PWD, 3) . '/vendor/autoload.php')) {
+    /** @noinspection PhpIncludeInspection */
+    require_once $autoloaderFilePath;
+} else {
+    throw new RuntimeException('Unable to find the Composer autoloader.');
+}
+
+// Include Annotation Processors for Buphalo
+require_once($PWD . '/src/AnnotationProcessors/DAO.php');
+require_once($PWD . '/src/AnnotationProcessors/DAOInterface.php');
+require_once($PWD . '/src/AnnotationProcessors/DAO/Builder.php');
+
+// Add exec envs
+putenv("Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__SourceDirectoryPath=$PWD/src");
+putenv("Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__FabricationDirectoryPath=$PWD/fab");
+putenv("Neighborhoods_Buphalo_V1_TargetApplication_BuilderInterface__NamespacePrefix=Neighborhoods\\BuphaloFitness\\Demo3");
+putenv("Neighborhoods_Buphalo_V1_TemplateTree_Map_Builder_FactoryInterface__TemplateTreeDirectoryPaths=$PWD/templates");
+
+// Include Buphalo
+require_once($PWD . '/../../../bin/v1/buphalo.php');
+
+// Diff the two directories
+exec("diff -r $PWD/control/ $PWD/fab/", $output, $return);
+
+if ($return) {
+    die(implode(PHP_EOL, $output) . PHP_EOL);
```

Comment:
consider `sprintf`
```suggestion
    die(sprintf('%s%s', implode(PHP_EOL, $output), PHP_EOL));
```

Reply:
Same as above. Adding a newline at the end feels clearer with concatenation to me.